virt/release_notes/virt-4-17-release-notes.adoc

```diff
@@ -79,6 +79,10 @@ Overcommitting memory on a highly utilized system can decrease workload performa
 //CNV-30590 NEW GA Host-side USB passthrough
 * As a cluster administrator, you can xref:../../virt/virtual_machines/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc#virt-configuring-usb-host-passthrough[expose USB devices in a cluster], making them available for virtual machine (VM) owners to assign to VMs. You expose a USB device by first enabling host passthrough and then configuring the VM to access the USB device.
 
+//CNV-40005 Application-aware resource quota
+* You can now use the Application-Aware Quota (AAQ) Operator to customize and manage resource quotas for individual components in an {product-title} cluster. The AAQ Operator provides the `ApplicationAwareResourceQuota` and `ApplicationAwareClusterResourceQuota` custom resource definitions (CRDs) that can be used to allocate resources without interfering with cluster-level activities such as upgrades and node maintenance.
```

Comment:
Is "AAQ" an accepted product abbreviation?

Reply:
I think it is an Operator abbreviation much like the ones listed here: https://docs.openshift.com/container-platform/4.16/virt/about_virt/virt-architecture.html

Is there a list somewhere of accepted abbreviations? @stu-gott Any thoughts?

Reply:
AAQ is the accepted product abbreviation. I understand the concern, @jeana-redhat, as the fully spelled out name has a "resource" snuck somewhere in there. It's been referred to officially as AAQ since late 2023.

Reply:
Yeah, if this is official no worries - I only ask because I am the source of an unofficial one (when I was new) that I guess has since become accepted :sweat_smile: I have heard since that we aren't supposed to be making initialisms willy-nilly ourselves, so I try to ask about any new ones I see :slightly_smiling_face:

Reply:
oh yeah, and @sjhala-ccs yeah there is a giant spreadsheet here: https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit?gid=1375785039#gid=1375785039

Reply:
Thank you, @jeana-redhat and @stu-gott!